### ISSUE TITLE
Could not upgrade Rocket.Chat server version 3.3.0 to version 3.4.1 due to database migration problem

### ISSUE THREAD

**Issue by rzaSF** [RocketChat/Rocket.Chat#18142]
### Description:

Attempted to upgrade from Rocket.Chat server 3.3.0 to 3.4.1, and had database migration failures.

### Steps to reproduce:

1. Install Rocket.Chat server 3.3.0
2. Execute Rocket.Chat update i.e.  `rocketchatctl update`

### Server Setup Information:

```
Jul  3 08:21:21 inst01 rocketchat[1409]: Using FileSystem for custom sounds storage
Jul  3 08:21:21 inst01 rocketchat[1409]: Using FileSystem for custom emoji storage
Jul  3 08:21:22 inst01 rocketchat[1409]: Browserslist: caniuse-lite is outdated. Please run next command `npm update`
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ System ➔ startup
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ +----------------------------------------------------+
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |                   SERVER RUNNING                   |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ +----------------------------------------------------+
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |                                                    |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |  Rocket.Chat Version: 3.3.0                        |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |       NodeJS Version: 12.17.0 - x64                |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |      MongoDB Version: 4.0.3                        |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |       MongoDB Engine: mmapv1                       |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |             Platform: linux                        |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |         Process Port: 3000                         |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |             Site URL: https://xxxxx.xxxxxxxxx.tld  |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |     ReplicaSet OpLog: Enabled                      |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |          Commit Hash: a987295718                   |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |        Commit Branch: HEAD                         |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ |                                                    |
Jul  3 08:21:22 inst01 rocketchat[1409]: ➔ +----------------------------------------------------+

```

- Version of Rocket.Chat Server: 3.3.0
- Operating System: Ubuntu 18.04.4 LTS
- Deployment Method: Digital Ocean droplet
- Number of Running Instances: 1
- DB Replicaset Oplog: (see above)
- NodeJS Version: (see above)
- MongoDB Version: (see above)

### Client Setup Information

- Desktop App or Browser Version:
- Operating System:

### Additional context

Server has always been updated using `rocketchatctl update`.  This instance was initially deployed with Rocket.Chat 2.4.9 (Digital Ocean droplet default), and has been upgraded through multiple versions over time to 3.3.0.

### Relevant logs:

Performed `tail -f /var/log/syslog &` before running rocketchatctl update process.
```
root@inst01:~# rocketchatctl update
Updates could be risky, you can use the backup option # rocketchatctl backup, to create a backup of the rocketchat database first. Please check that you have enough space in your system to store the backup.
Current update available for RocketChat server: from 3.3.0 to 3.4.1commit.
Jul  3 08:22:08 inst01 systemd[1]: Stopping The Rocket.Chat server...
Jul  3 08:22:08 inst01 systemd[1]: Stopped The Rocket.Chat server.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   132  100   132    0     0   1859      0 --:--:-- --:--:-- --:--:--  1859
100  154M  100  154M    0     0   199M      0 --:--:-- --:--:-- --:--:--  199M
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

> fibers@4.0.3 install /tmp/bundle/programs/server/node_modules/fibers
> node build.js || nodejs build.js

`linux-x64-72-glibc` exists; testing
Binary is fine; exiting
npm WARN lifecycle meteor-dev-bundle@~install: cannot run in wd meteor-dev-bundle@ node npm-rebuild.js (wd=/tmp/bundle/programs/server)
added 146 packages from 122 contributors and audited 147 packages in 6.228s

1 package is looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Jul  3 08:22:28 inst01 systemd[1]: Started The Rocket.Chat server.
Waiting up to 60 seconds for RocketChat server to be active ... 2
Waiting up to 60 seconds for RocketChat server to be active ... 4
Waiting up to 60 seconds for RocketChat server to be active ... 6
Waiting up to 60 seconds for RocketChat server to be active ... 8
Waiting up to 60 seconds for RocketChat server to be active ... 10
Waiting up to 60 seconds for RocketChat server to be active ... 12
Waiting up to 60 seconds for RocketChat server to be active ... 14
Waiting up to 60 seconds for RocketChat server to be active ... 16
Waiting up to 60 seconds for RocketChat server to be active ... 18
Jul  3 08:22:48 inst01 rocketchat[1578]: Setting default file store to FileSystem
Waiting up to 60 seconds for RocketChat server to be active ... 20
Waiting up to 60 seconds for RocketChat server to be active ... 22
Waiting up to 60 seconds for RocketChat server to be active ... 24
Jul  3 08:22:55 inst01 rocketchat[1578]: {"line":"120","file":"migrations.js","message":"Migrations: Migrating from version 191 -> 197","time":{"$date":1593764575085},"level":"info"}
Jul  3 08:22:55 inst01 rocketchat[1578]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 192","time":{"$date":1593764575089},"level":"info"}
Jul  3 08:22:55 inst01 rocketchat[1578]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 193","time":{"$date":1593764575127},"level":"info"}
Jul  3 08:22:55 inst01 rocketchat[1578]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 194","time":{"$date":1593764575137},"level":"info"}
Jul  3 08:22:55 inst01 rocketchat[1578]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 195","time":{"$date":1593764575158},"level":"info"}
Waiting up to 60 seconds for RocketChat server to be active ... 26
Jul  3 08:22:55 inst01 rocketchat[1578]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 196","time":{"$date":1593764575419},"level":"info"}
Jul  3 08:22:55 inst01 rocketchat[1578]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 197","time":{"$date":1593764575433},"level":"info"}
Jul  3 08:22:55 inst01 rocketchat[1578]: {"line":"120","file":"migrations.js","message":"Migrations: Finished migrating.","time":{"$date":1593764575449},"level":"info"}
Waiting up to 60 seconds for RocketChat server to be active ... 28
Waiting up to 60 seconds for RocketChat server to be active ... 30
Waiting up to 60 seconds for RocketChat server to be active ... 32
Waiting up to 60 seconds for RocketChat server to be active ... 34
Jul  3 08:23:05 inst01 rocketchat[1578]: Not migrating, control is locked. Attempt 1/30. Trying again in 10 seconds.
Jul  3 08:23:05 inst01 rocketchat[1578]: {"line":"120","file":"migrations.js","message":"Migrations: Not migrating, already at version 197","time":{"$date":1593764585459},"level":"info"}
Waiting up to 60 seconds for RocketChat server to be active ... 36
Waiting up to 60 seconds for RocketChat server to be active ... 38
Waiting up to 60 seconds for RocketChat server to be active ... 40
Waiting up to 60 seconds for RocketChat server to be active ... 42
Waiting up to 60 seconds for RocketChat server to be active ... 44
Waiting up to 60 seconds for RocketChat server to be active ... 46
Waiting up to 60 seconds for RocketChat server to be active ... 48
Waiting up to 60 seconds for RocketChat server to be active ... 50
Waiting up to 60 seconds for RocketChat server to be active ... 52
Waiting up to 60 seconds for RocketChat server to be active ... 54
Waiting up to 60 seconds for RocketChat server to be active ... 56
Waiting up to 60 seconds for RocketChat server to be active ... 58
Jul  3 08:23:29 inst01 rocketchat[1578]: Updating process.env.MAIL_URL
Jul  3 08:23:29 inst01 rocketchat[1578]: Loaded the Apps Framework and loaded a total of 3 Apps!
Jul  3 08:23:29 inst01 rocketchat[1578]: Using FileSystem for custom sounds storage
Jul  3 08:23:29 inst01 rocketchat[1578]: Using FileSystem for custom emoji storage
Jul  3 08:23:29 inst01 rocketchat[1578]: Browserslist: caniuse-lite is outdated. Please run next command `npm update`
Waiting up to 60 seconds for RocketChat server to be active ... 60
Jul  3 08:23:33 inst01 systemd[1]: Stopping The Rocket.Chat server...
Error in updated RocketChat server health check, restoring backup.
root@inst01:~# Jul  3 08:23:33 inst01 systemd[1]: Stopped The Rocket.Chat server.
Jul  3 08:23:33 inst01 systemd[1]: Started The Rocket.Chat server.
Jul  3 08:23:54 inst01 rocketchat[1982]: Setting default file store to FileSystem
Jul  3 08:24:03 inst01 rocketchat[1982]: Exception in callback of async function: errorClass [Error]: [Can't find migration version 197]
Jul  3 08:24:03 inst01 rocketchat[1982]:     at Object.Migrations._findIndexByVersion (app/migrations/server/migrations.js:402:8)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at Object.Migrations._migrateTo (app/migrations/server/migrations.js:254:24)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at Object.Migrations.migrateTo (app/migrations/server/migrations.js:179:20)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at module (server/startup/migrations/xrun.js:6:13)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at fileEvaluate (packages/modules-runtime.js:336:7)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at Module.require (packages/modules-runtime.js:238:14)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at Module.moduleLink [as link] (/opt/Rocket.Chat/programs/server/npm/node_modules/meteor/modules/node_modules/reify/lib/runtime/index.js:52:22)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at module (server/startup/migrations/index.js:1:4188)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at fileEvaluate (packages/modules-runtime.js:336:7)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at Module.require (packages/modules-runtime.js:238:14)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at Module.moduleLink [as link] (/opt/Rocket.Chat/programs/server/npm/node_modules/meteor/modules/node_modules/reify/lib/runtime/index.js:52:22)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at module (server/main.js:1:231)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at fileEvaluate (packages/modules-runtime.js:336:7)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at Module.require (packages/modules-runtime.js:238:14)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at require (packages/modules-runtime.js:258:21)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at /opt/Rocket.Chat/programs/server/app/app.js:176583:15
Jul  3 08:24:03 inst01 rocketchat[1982]:     at /opt/Rocket.Chat/programs/server/boot.js:401:38
Jul  3 08:24:03 inst01 rocketchat[1982]:     at Array.forEach (<anonymous>)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at /opt/Rocket.Chat/programs/server/boot.js:226:21
Jul  3 08:24:03 inst01 rocketchat[1982]:     at /opt/Rocket.Chat/programs/server/boot.js:464:7
Jul  3 08:24:03 inst01 rocketchat[1982]:     at Function.run (/opt/Rocket.Chat/programs/server/profile.js:280:14)
Jul  3 08:24:03 inst01 rocketchat[1982]:     at /opt/Rocket.Chat/programs/server/boot.js:463:13 {
Jul  3 08:24:03 inst01 rocketchat[1982]:   isClientSafe: true,
Jul  3 08:24:03 inst01 rocketchat[1982]:   error: "Can't find migration version 197",
Jul  3 08:24:03 inst01 rocketchat[1982]:   reason: undefined,
Jul  3 08:24:03 inst01 rocketchat[1982]:   details: undefined,
Jul  3 08:24:03 inst01 rocketchat[1982]:   errorType: 'Meteor.Error'
Jul  3 08:24:03 inst01 rocketchat[1982]: }

```

**Comment by Bruce-Ming** [comment]
Modify the mongodb database document migratrions locked to false

**Comment by Bruce-Ming** [comment]
migration version is  wrong，Different versions of rocketchat correspond to different migration versions，3.4.1->197，It seems that your database has been upgraded, but the service has not been upgraded，So Can't find migration version 197 file

**Comment by rzaSF** [comment]
I'm not sure I understand how the DB was updated, unless it happened from 3.3.0?  I hadn't upgraded to any intermediate version between 3.3.0 and 3.4.1.

So what would be the recommendation to resolve this?  Is there a way to verify the DB state?  (sorry I'm not very versed in mongo)

**Comment by rzaSF** [comment]
Hunting around, I found `db.migrations.find()` ... this is what is in my migrations collection:

```
rs01:PRIMARY> db.migrations.find()
{ "_id" : "control", "locked" : false, "version" : 191, "buildAt" : "2020-05-28T02:23:09.159Z", "lockedAt" : ISODate("2020-07-03T09:16:28.522Z") }
```

**Comment by Bruce-Ming** [comment]
**Error in updated RocketChat server health check, restoring backup，**  It seems that the upgrade failed

**Comment by Bruce-Ming** [comment]
The easiest way is to back up your mongodb database data, and then reinstall rocketchat to restore the data

**Comment by rzaSF** [comment]
> The easiest way is to back up your mongodb database data, and then reinstall rocketchat to restore the data

That sounds like, "Upgrade is broken in 3.4.1 (coming from 3.3.0).  Install a fresh copy and import the backup."

**Comment by rzaSF** [comment]
Well, I went for it again, from 3.3.0...  and voila, it worked.  I'm wondering if something was making migration take too long?  Didn't change anything, simply started from the same snapshot with 3.3.0 as before (as I had reproduced at least 5 times).

TL;DR It's working.

```
<<SNIP>>
2020-07-04T04:57:50.190+0000	done dumping rocketchat.rocketchat_livechat_unit_monitors (0 documents)
2020-07-04T04:57:50.190+0000	writing rocketchat.rocketchat_livechat_department to archive on stdout
2020-07-04T04:57:50.196+0000	done dumping rocketchat.meteor_oauth_pendingRequestTokens (0 documents)
2020-07-04T04:57:50.197+0000	writing rocketchat.rocketchat_livechat_priority to archive on stdout
2020-07-04T04:57:50.204+0000	done dumping rocketchat.rocketchat_webdav_accounts (0 documents)
2020-07-04T04:57:50.205+0000	done dumping rocketchat.rocketchat_reports (0 documents)
2020-07-04T04:57:50.206+0000	done dumping rocketchat.rocketchat_livechat_department (0 documents)
2020-07-04T04:57:50.206+0000	done dumping rocketchat.rocketchat_livechat_priority (0 documents)
Jul  4 04:57:50 inst01 systemd[1]: Started The Rocket.Chat server.
Backup done successfully, check for the backup file in /tmp/dump_1593838666.gz. You can restore using # mongorestore --host localhost:27017 --db rocketchat --archive --gzip < /tmp/dump_1593838666.gz
root@inst01:~# rocketchatctl update
Updates could be risky, you can use the backup option # rocketchatctl backup, to create a backup of the rocketchat database first. Please check that you have enough space in your system to store the backup.
/usr/local/bin/rocketchatctl: line 900: [: 3.4.1commit: unary operator expected
Could not determine if updates available for RocketChat server.
root@inst01:~# rocketchatctl check-updates
Updates could be risky, you can use the backup option # rocketchatctl backup, to create a backup of the rocketchat database first. Please check that you have enough space in your system to store the backup.
/usr/local/bin/rocketchatctl: line 900: [: 3.4.1commit: unary operator expected
Could not determine if updates available for RocketChat server.
root@inst01:~# Jul  4 04:58:08 inst01 rocketchat[1370]: Setting default file store to FileSystem
Jul  4 04:58:13 inst01 rocketchat[1370]: {"line":"120","file":"migrations.js","message":"Migrations: Not migrating, already at version 191","time":{"$date":1593838693744},"level":"info"}

root@inst01:~# rocketchatctl update
Updates could be risky, you can use the backup option # rocketchatctl backup, to create a backup of the rocketchat database first. Please check that you have enough space in your system to store the backup.
/usr/local/bin/rocketchatctl: line 900: [: 3.4.1commit: unary operator expected
Could not determine if updates available for RocketChat server.
root@inst01:~# rocketchatctl updateJul  4 04:58:34 inst01 rocketchat[1370]: Updating process.env.MAIL_URL
Jul  4 04:58:34 inst01 rocketchat[1370]: Loaded the Apps Framework and loaded a total of 3 Apps!
Jul  4 04:58:34 inst01 rocketchat[1370]: Using FileSystem for custom sounds storage
Jul  4 04:58:34 inst01 rocketchat[1370]: Using FileSystem for custom emoji storage
Jul  4 04:58:35 inst01 rocketchat[1370]: Browserslist: caniuse-lite is outdated. Please run next command `npm update`
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ System ➔ startup
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ +----------------------------------------------------+
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |                   SERVER RUNNING                   |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ +----------------------------------------------------+
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |                                                    |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |  Rocket.Chat Version: 3.3.0                        |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |       NodeJS Version: 12.17.0 - x64                |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |      MongoDB Version: 4.0.3                        |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |       MongoDB Engine: mmapv1                       |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |             Platform: linux                        |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |         Process Port: 3000                         |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |             Site URL: https://xxxxx.xxxxxxxxx.tld  |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |     ReplicaSet OpLog: Enabled                      |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |          Commit Hash: a987295718                   |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |        Commit Branch: HEAD                         |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ |                                                    |
Jul  4 04:58:35 inst01 rocketchat[1370]: ➔ +----------------------------------------------------+

Updates could be risky, you can use the backup option # rocketchatctl backup, to create a backup of the rocketchat database first. Please check that you have enough space in your system to store the backup.
Current update available for RocketChat server: from 3.3.0 to 3.4.1commit.
Jul  4 04:58:39 inst01 systemd[1]: Stopping The Rocket.Chat server...
Jul  4 04:58:39 inst01 systemd[1]: Stopped The Rocket.Chat server.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   132  100   132    0     0   2030      0 --:--:-- --:--:-- --:--:--  2030
100  154M  100  154M    0     0   175M      0 --:--:-- --:--:-- --:--:--  224M
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

> fibers@4.0.3 install /tmp/bundle/programs/server/node_modules/fibers
> node build.js || nodejs build.js

`linux-x64-72-glibc` exists; testing
Binary is fine; exiting
npm WARN lifecycle meteor-dev-bundle@~install: cannot run in wd meteor-dev-bundle@ node npm-rebuild.js (wd=/tmp/bundle/programs/server)
added 146 packages from 122 contributors and audited 147 packages in 5.173s

1 package is looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Jul  4 04:58:57 inst01 systemd[1]: Started The Rocket.Chat server.
Waiting up to 60 seconds for RocketChat server to be active ... 2
Waiting up to 60 seconds for RocketChat server to be active ... 4
Waiting up to 60 seconds for RocketChat server to be active ... 6
Waiting up to 60 seconds for RocketChat server to be active ... 8
Waiting up to 60 seconds for RocketChat server to be active ... 10
Waiting up to 60 seconds for RocketChat server to be active ... 12
Waiting up to 60 seconds for RocketChat server to be active ... 14
Waiting up to 60 seconds for RocketChat server to be active ... 16
Jul  4 04:59:14 inst01 rocketchat[1598]: Setting default file store to FileSystem
Waiting up to 60 seconds for RocketChat server to be active ... 18
Waiting up to 60 seconds for RocketChat server to be active ... 20
Waiting up to 60 seconds for RocketChat server to be active ... 22
Jul  4 04:59:21 inst01 rocketchat[1598]: {"line":"120","file":"migrations.js","message":"Migrations: Migrating from version 191 -> 197","time":{"$date":1593838761128},"level":"info"}
Jul  4 04:59:21 inst01 rocketchat[1598]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 192","time":{"$date":1593838761132},"level":"info"}
Jul  4 04:59:21 inst01 rocketchat[1598]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 193","time":{"$date":1593838761164},"level":"info"}
Jul  4 04:59:21 inst01 rocketchat[1598]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 194","time":{"$date":1593838761173},"level":"info"}
Jul  4 04:59:21 inst01 rocketchat[1598]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 195","time":{"$date":1593838761190},"level":"info"}
Jul  4 04:59:21 inst01 rocketchat[1598]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 196","time":{"$date":1593838761351},"level":"info"}
Jul  4 04:59:21 inst01 rocketchat[1598]: {"line":"120","file":"migrations.js","message":"Migrations: Running up() on version 197","time":{"$date":1593838761363},"level":"info"}
Jul  4 04:59:21 inst01 rocketchat[1598]: {"line":"120","file":"migrations.js","message":"Migrations: Finished migrating.","time":{"$date":1593838761375},"level":"info"}
Waiting up to 60 seconds for RocketChat server to be active ... 24
Waiting up to 60 seconds for RocketChat server to be active ... 26
Waiting up to 60 seconds for RocketChat server to be active ... 28
Waiting up to 60 seconds for RocketChat server to be active ... 30
Waiting up to 60 seconds for RocketChat server to be active ... 32
Jul  4 04:59:31 inst01 rocketchat[1598]: Not migrating, control is locked. Attempt 1/30. Trying again in 10 seconds.
Jul  4 04:59:31 inst01 rocketchat[1598]: {"line":"120","file":"migrations.js","message":"Migrations: Not migrating, already at version 197","time":{"$date":1593838771384},"level":"info"}
Waiting up to 60 seconds for RocketChat server to be active ... 34
Waiting up to 60 seconds for RocketChat server to be active ... 36
Waiting up to 60 seconds for RocketChat server to be active ... 38
Waiting up to 60 seconds for RocketChat server to be active ... 40
Waiting up to 60 seconds for RocketChat server to be active ... 42
Waiting up to 60 seconds for RocketChat server to be active ... 44
Waiting up to 60 seconds for RocketChat server to be active ... 46
Waiting up to 60 seconds for RocketChat server to be active ... 48
Waiting up to 60 seconds for RocketChat server to be active ... 50
Waiting up to 60 seconds for RocketChat server to be active ... 52
Jul  4 04:59:52 inst01 rocketchat[1598]: Updating process.env.MAIL_URL
Waiting up to 60 seconds for RocketChat server to be active ... 54
Jul  4 04:59:53 inst01 rocketchat[1598]: Loaded the Apps Framework and loaded a total of 3 Apps!
Jul  4 04:59:53 inst01 rocketchat[1598]: Using FileSystem for custom sounds storage
Jul  4 04:59:53 inst01 rocketchat[1598]: Using FileSystem for custom emoji storage
Jul  4 04:59:53 inst01 rocketchat[1598]: Browserslist: caniuse-lite is outdated. Please run next command `npm update`
Waiting up to 60 seconds for RocketChat server to be active ... 56
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ System ➔ startup
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ +----------------------------------------------------+
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |                   SERVER RUNNING                   |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ +----------------------------------------------------+
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |                                                    |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |  Rocket.Chat Version: 3.4.1                        |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |       NodeJS Version: 12.17.0 - x64                |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |      MongoDB Version: 4.0.3                        |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |       MongoDB Engine: mmapv1                       |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |             Platform: linux                        |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |         Process Port: 3000                         |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |             Site URL: https://xxxxx.xxxxxxxxx.tld  |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |     ReplicaSet OpLog: Enabled                      |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |          Commit Hash: 21157c0c4f                   |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |        Commit Branch: HEAD                         |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ |                                                    |
Jul  4 04:59:55 inst01 rocketchat[1598]: ➔ +----------------------------------------------------+
Waiting up to 60 seconds for RocketChat server to be active ... 58
Waiting up to 60 seconds for RocketChat server to be active ... 60
RocketChat server updated to latest version :)

```
